carma/launch/environment.launch

```diff
@@ -111,6 +115,10 @@
   
   <!-- lidar_euclidean_cluster_detect -->
   <group>
+  <remap to="$(optenv CARMA_ENV_NS)/detection/lidar_detector/cloud_clusters" from="/detection/lidar_detector/cloud_clusters"/>
+  <remap to="$(optenv CARMA_ENV_NS)/detection/lidar_detector/objects" from="/detection/lidar_detector/objects"/>
+  <remap to="$(optenv CARMA_ENV_NS)/detection/lidar_detector/objects_markers" from="/detection/lidar_detector/objects_markers"/>
```

Comment:
is the leading $(optenv CARMA_ENV_NS)/ needed here since that is the namespace of this file?

Reply:
Also is the objects_markers here the one we see in rviz? Do we need to update the rviz config as well?

Reply:
changed object_markers in rviz, and left todo comment on  the ENV_NS as discussed offline